broker-cli/commands/order/summary.js

```diff
@@ -49,11 +64,34 @@ function createUI (market, orders) {
  * @param {Logger} logger
  */
 async function summary (args, opts, logger) {
-  const { market, rpcAddress } = opts
-  const request = { market }
+  const { market } = args
```

Comment:
re: our chat in person, `market` is still technically an option even though it's required.

Reply:
Yup, will change this back. Thanks @bradleystachurski